src/commands/pczt/inspect.rs

```diff
@@ -359,6 +392,40 @@ impl Command {
                         u32::from(*idx)
                     );
                 }
+
+                // Display decrypted note information if available
+                if let Some((note, recipient, memo_bytes, scope)) = decrypted {
+                    println!("  - Decrypted note:");
+                    println!("    Value: {} zatoshis", note.value().inner());
+                    print_verified_recipient_address(
+                        recipient,
+                        orchard_fvk.as_ref().unwrap(),
+                        *scope,
+                    );
```

Comment:
Printing out the the recipient and value should already be covered if they have not been redacted. We'll need an intermediate step. 

1. collect the data 
2. figure out what we have to print without duplication
3. do the printing